mtags/src/main/scala/scala/meta/internal/pc/Completions.scala

```diff
@@ -336,7 +336,7 @@ trait Completions { this: MetalsGlobal =>
   // variable but it avoids repeating traversals from the compiler
   // implementation of `completionsAt(pos)`.
   var lastVisistedParentTrees: List[Tree] = Nil
-  sealed abstract class CompletionPosition {
+  abstract class CompletionPosition {
```

Comment:
Removed a `sealed` modifier to split `Scaladoc` related logic into another file.
I think we cannot avoid removing this for splitting each CompletionPosition logics into other files, and I think it's ok with removing this modifier 🤔